compute_tools/src/bin/compute_ctl.rs

```diff
@@ -482,6 +480,8 @@ fn cli() -> clap::Command {
                 .value_name("FILECACHE_CONNSTR"),
         )
         .arg(
+            // DEPRECATED, NO LONGER DOES ANYTHING.
+            // See https://github.com/neondatabase/cloud/issues/7516
             Arg::new("file-cache-on-disk")
                 .long("file-cache-on-disk")
                 .action(clap::ArgAction::SetTrue),
```

Comment:
If this could be used in some environment, could log a warning when true.

Reply:
Hm, yeah I was just thinking of this. Curious for your thoughts:

Currently the control plane always sets this argument (neondatabase/cloud#7522), then once this PR is released, we'll change the control plane to never set this argument  (neondatabase/cloud#7523).

So, _right now_ it'd be good to have warnings for when this arg is _not_ set. But we'll end up with warnings once we make the control plane change anyways, so... I don't know.

Reply:
Aah well, if it's always set, then perhaps not. I was thinkin gif some tests shouldn't use this flag anymore, or if there were more conditions to using it. Feel free to close.